app/controllers/admin.js

```diff
@@ -0,0 +1,38 @@
+const jwt = require('jwt-simple');
+const { logger } = require('express-wolox-logger');
+const { statusCodes, errors } = require('../helpers');
+const { adminMapper } = require('../mappers');
+const { UserService } = require('../services');
+
+const signUpAdmin = async (req, res, next) => {
+  try {
+    const token = req.header('token');
+    const tokenDecode = jwt.decode(token, process.env.JWT_KEY_SECRET);
+    const userInfo = await UserService.getUser({ id: tokenDecode.id });
+    if (
+      tokenDecode.firstName !== userInfo.firstName ||
+      tokenDecode.lastName !== userInfo.lastName ||
+      tokenDecode.role !== userInfo.role
```

Comment:
put these conditions inside an array and then

```suggestion
      conditions.some(condition => !!condition)
```